channeld/channeld.c

```diff
@@ -165,8 +173,9 @@ static void billboard_update(const struct peer *peer)
 	if (peer->funding_locked[LOCAL] && peer->funding_locked[REMOTE])
 		funding_status = "Funding transaction locked.";
 	else if (!peer->funding_locked[LOCAL] && !peer->funding_locked[REMOTE])
-		/* FIXME: Say how many blocks to go! */
-		funding_status = "Funding needs more confirmations.";
+		funding_status = tal_fmt(tmpctx,
+					"Funding needs %d confirmations.",
```

Comment:
Finally, can I suggest a subtle change:
`Funding needs %d confirmations.` -> `Funding needs %d confirmations before lockin.`

Hopefully this makes it also more clear that a lockin is not reversed with a reorg.

Reply:
Nice descripthion! :)